src/coreclr/vm/jithelpers.cpp

```diff
@@ -1526,8 +1531,12 @@ HCIMPL1(void, IL_Throw,  Object* obj)
     {
         Thread *pThread = GetThread();
 
+#ifdef TARGET_X86
+        SoftwareExceptionFrame exceptionFrame(transitionBlock);
```

Comment:
A nit - can we keep the same empty constructor as on the other architectures and on x86 just instead of the `RtlCaptureContext` use a new function that would do what we do in the new constructor? To keep it more similar for the architectures?

Reply:
Yes, will do.